assets/components/account-menu/README.md

```diff
@@ -1,11 +1,6 @@
 # Account Menu
 
-<div class="alert alert--info">
-
-<p class="alert__message">This pattern is work in progress.</p>
-<p class="alert__message">View the todo list for this pattern on [GitHub](https://github.com/hmrc/design-patterns/issues/128).</p>
-
-</div>
+{{ wip('https://github.com/hmrc/design-patterns/issues/128') }}
```

Comment:
We only need the issue number supplied here.

All issues will be on the same repo and if it changes it'd be easier to update it in the macro rather than all READMEs.